### ISSUE TITLE
Mockup

### ISSUE THREAD

**Issue by muskuloes** [cjwfly1988/firstStep#1]
It would be nice to have mockups of how the webpage should like.

**Comment by cjwfly1988** [comment]
Sure, I will do it.

**Comment by cjwfly1988** [comment]
@Muskuloese I looked up something, maybe this can be an idea to start with 
https://wordpress.org/themes/hoot-du/

**Comment by muskuloes** [comment]
yeah, but that is more for a gallery kind of website. I guess we just want to have show some information text.

For a beginning we can just have as a minimum product a button that when click requests and displays data and then we can adapt the view.

**Comment by cjwfly1988** [comment]
I know, I want to somehow put those images there in different categories with links. 
Let's do the backend thing with GO since I am learning it.
